l10n_in_account_tax/__openerp__.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2012-Today Serpent Consulting Services Pvt. Ltd.
+#    (<http://www.serpentcs.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>
+#
+############################################################################
+
+{
+    'name': 'Indian Tax Category',
+    'version': '1.0',
+    'category': 'Indian Localization',
+    'description': """Configure Indian Tax with Category
+========================================================================
+- Added tax category on tax to determine different types of taxes during
```

Comment:
The description should start with something like.. 'This module ads..., This modules does..'
It should not appear to be the commit message.